index.js

```diff
@@ -12,14 +12,23 @@ module.exports = (css, settings) => {
 
   // Add the directory containing the current file to includePaths to enable relative
   // imports, only works when the filename is provided
-  const includePaths = settings.sassOptions && settings.sassOptions.includePaths || []
+  const includePaths = settings.sassOptions && settings.sassOptions.includePaths || [];
+  
   if (settings.babel && settings.babel.filename) {
     includePaths.push(path.dirname(settings.babel.filename));
   }
 
-  const preprocessed = sass.renderSync(Object.assign({
-    data: cssWithPlaceholders
-  }, settings.sassOptions, { includePaths })).css.toString()
+  // Prepend option data to cssWithPlaceholders
+  const optionData = settings.sassOptions && settings.sassOptions.data || "";
+  const data = optionData + "\n" + cssWithPlaceholders;
```

Comment:
The line break here is needed, otherwise `node-sass` won't parse the final object